items/item.go

```diff
@@ -4,27 +4,30 @@ import (
 	"errors"
 	"fmt"
 	"io"
+	"log"
 	"strconv"
 	"strings"
+	"sync"
 
 	"github.com/ndlib/bendo/store"
 )
 
 // A Store holds a collection of items
 type Store struct {
-	cache ItemCache
-	S     store.Store // the underlying bundle store
+	cache    ItemCache
+	S        store.Store // the underlying bundle store
+	useStore bool        // true - use bundlestore: false - use only itemCache
```

Comment:
FYI, `useStore` needs a mutex to serialize access. But since it will mostly be read, I don't think we need to add it right now.